### ISSUE TITLE
'onion skin' for levels

### ISSUE THREAD

**Issue by IgnacioFer** [tahoma2d/tahoma2d#741]
I love the onion skin in OpTz and T2D, but I miss an 'onion skin' feature for the levels. This would not be a real onion skin but a way to easily see which level you are on and which elements are on different levels, so you don't end up drawing an element in the wrong level (which I do all the time)

It'd be great to have an extra toggle in the timeline, somewhere near the 'new level' button or the level visibility and lock toggles, which when toggled on makes all levels except the one you are on blue (colour to be chosen from menu like individual level colour filters and/or customisable in preferences). 

Unlike with the onion skin, I don't think it would serve any purpose to differentiate which levels are closer or farther from the selected level by having different degrees of opacity, so basically all this toggle would do would be to apply a level colour filter to all levels other than the one currently selected.

This would be a lot easier than having to go to individual levels and toggle individual colour filters on/off.

Thanks!

**Comment by beeheemooth** [comment]
I agree with that. At the moment, the opposite is happening. You need to focus on one level and see the rest of the levels just in context, but for this you need to add transparency (or enable a filter) to all the other columns - this is awful! At the same time, you cannot use even such a cumbersome approach while in the sub-scene.
 To all this, I would also add the strange logic of the "Ink Only" option. When you draw overlapping vector layers and want to see where the line break occurs, enabling the "Ink Only" option only works for the current column. If the lower columns are dark in color, then you will not see anything, by clicking with the control point tool you will mistakenly select other columns, while the option will switch to this column, and the upper one will become opaque. Aaaarrrrgggghhh!
I will add to this the nightmare of what happens with the Hide / Show columns options (I will not repeat myself, I wrote about this for a long time).

Sometimes it seems to me that if all these options worked the other way around, it would be much better, hehe.

**Comment by artisteacher** [comment]
This isn't as easy as a toggle, but you can add color filters in the Timeline to your levels for easier identification.

**Comment by IgnacioFer** [comment]
It could be something like this:
![level_onion_skin](https://user-images.githubusercontent.com/88771454/129348467-2a26749c-8390-4058-9406-62eb50d2105c.jpg)

The problem with adding colour filters manually is that when animating on several levels you are typically jumping from one level to another, so manually resetting the filters is not really practical. When toggled on, this would automatically recognize which level is selected and apply the colour filter to all other levels.

**Comment by artisteacher** [comment]
I agree that the color filters aren't as helpful as what you describe. Sounds like a great idea to me.
